### PR TITLE
Update configure-email-forwarding.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the related content with link on how to control external email forwarding as it is related to this document.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6161

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/email/configure-email-forwarding.md [hunk 1]
```diff
@@ -113,3 +113,4 @@ You must be an Exchange administrator or Global administrator in Microsoft 365 t
 [Create a shared mailbox](../email/create-a-shared-mailbox.md) (article)\
 [Send email from a different address](https://support.microsoft.com/office/ccba89cb-141c-4a36-8c56-6d16a8556d2e) (article)\
 [Change a user name and email address](../add-users/change-a-user-name-and-email-address.md) (article)
+[control external email forwarding](/security/office-365-security/external-email-forwarding) (article)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/configure-email-forwarding.md]:
```suggestion
[Control automatic external email forwarding in Microsoft 365](/security/office-365-security/external-email-forwarding) (article)
```

File: microsoft-365/admin/email/configure-email-forwarding.md [hunk 2]
```diff
@@ -113,3 +113,4 @@ You must be an Exchange administrator or Global administrator in Microsoft 365 t
 [Create a shared mailbox](../email/create-a-shared-mailbox.md) (article)\
 [Send email from a different address](https://support.microsoft.com/office/ccba89cb-141c-4a36-8c56-6d16a8556d2e) (article)\
 [Change a user name and email address](../add-users/change-a-user-name-and-email-address.md) (article)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/configure-email-forwarding.md]:
```suggestion
[Change a user name and email address](../add-users/change-a-user-name-and-email-address.md) (article)\
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**